docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+# Deploying to AWS
+
+!!! danger "AWS Admin Privileges Required"
+
+    This deployment can only be run by AWS administrators, so we assume the user has a basic understanding of code
```

Comment:
Suggested text change:

```
!!! danger "Prerequisites"
This deployment can only be run by users with AWS administrator privileges, so contact your organization's cloud admins if you need access.  In addition, you should have a basic understanding of code development, Docker containers, and the AWS cloud.
```

Reply:
Good suggestion. I made that change to the admonition text.